<ui version="4.0" >
 <class>ImportMetadataWidget</class>
 <widget class="QWidget" name="ImportMetadataWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>261</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Import</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="labelUrl" >
     <property name="text" >
      <string>Source Directory:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="KUrlRequester" name="sourceLoc" />
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="repositoryLabel" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Maximum" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>&amp;Repository:</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
     <property name="buddy" >
      <cstring>repository</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="KLineEdit" name="repository" />
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="moduleLabel" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Maximum" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Mo&amp;dule:</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
     <property name="buddy" >
      <cstring>module</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="KLineEdit" name="module" />
   </item>
   <item row="3" column="0" >
    <widget class="QLabel" name="vendorLabel" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Maximum" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>&amp;Vendor tag:</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
     <property name="buddy" >
      <cstring>vendorTag</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <widget class="KLineEdit" name="vendorTag" >
     <property name="text" >
      <string>vendor</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" >
    <widget class="QLabel" name="releaseLabel" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Maximum" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Re&amp;lease tag:</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
     <property name="buddy" >
      <cstring>releaseTag</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1" >
    <widget class="KLineEdit" name="releaseTag" >
     <property name="text" >
      <string>start</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" >
    <widget class="QLabel" name="commentLabel" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Maximum" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Co&amp;mment:</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
     <property name="buddy" >
      <cstring>comment</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1" >
    <widget class="KTextEdit" name="comment" />
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KTextEdit</class>
   <extends>QTextEdit</extends>
   <header>ktextedit.h</header>
  </customwidget>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>repository</tabstop>
  <tabstop>module</tabstop>
  <tabstop>vendorTag</tabstop>
  <tabstop>releaseTag</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
